src/tests/dts_common.c

```diff
@@ -242,13 +242,13 @@ pool_init(struct dts_context *tsc)
  bcast:
 	if (tsc->tsc_mpi_size <= 1)
 		goto out; /* don't need to share handle */
-
-	MPI_Bcast(&rc, 1, MPI_INT, 0, MPI_COMM_WORLD);
-	if (rc)
+	if (!tsc->tsc_pmem_file)
+		MPI_Bcast(&rc, 1, MPI_INT, 0, MPI_COMM_WORLD);
+        if (rc)
```

Comment:
(style)  code indent should use tabs where possible

Reply:
This is from old version of daos.. ignoring this.